File: 2020/036-JuliaVsPython/lab.jl
# 343 s instead of 16 in javascript. Boring.
# julia --optimize=3 sudoku.js

using CPUTime

function execute()
	primes = []
	a = 100000000
	for j in a:a+100
		i=2
		k=1
		while k!=0 && i<j
			k=j%i
			i+=1
		end
		if i==j
			push!(primes,i)
		end
	end
	println(length(primes))
end

@time @CPUtime execute()

File: 2020/036-JuliaVsPython/lab.py
import time
import math

def execute():
	primes = []
	a = 100000000
	for j in range(a,a+100):
		i=2
		k=1
		while k!=0 and i<j:
			k=j%i
			i+=1
		if i==j:
			primes.append(i)
	print(len(primes))

start = time.time()
execute()
print(time.time()-start)
